src/gt4sd/conftest.py

```diff
@@ -22,11 +22,33 @@
 # SOFTWARE.
 #
 """Make pytest fixtures available to multiple test directories."""
+import atexit
+from contextlib import ExitStack
+from pathlib import PosixPath
 
+import importlib_resources
 import pytest
 
 
 @pytest.fixture
 def mock_wrong_s3_env(monkeypatch):
     """Changes an environment variable to break the s3 connection."""
     monkeypatch.setenv("GT4SD_S3_SECRET_KEY", "(╯°□°）╯︵ ┻━┻")
+
+
+def exitclose_file_creator(file_path: str) -> PosixPath:
```

Comment:
I would move this to `src/gt4sd/tests/utils.py` as this file was supposed to contain only pytest fixtures.